### DISCUSSION TITLE
Installation problem

### DISCUSSION THREAD

**Discussion by yzjalwind** [com-lihaoyi/mill#1422] [Q&A]
Hi, I want to know if mill is installed successfully and where is the problem
![image](https://user-images.githubusercontent.com/57080333/126853723-a80f763e-f0f2-4461-ae5d-ed7061e25220.png)

**Comment by lefou** [comment]
Can you show the output of `ls -la`?

**Reply by yzjalwind** [reply]
Please enter ls -la  in which directory ?

**Reply by yzjalwind** [reply]
![image](https://user-images.githubusercontent.com/57080333/126865600-0024d9f9-46a9-40e1-b12c-e1620a935562.png)
![image](https://user-images.githubusercontent.com/57080333/126865618-1d96ec6c-45ab-4b27-bf61-2e042700c809.png)

**Reply by lefou** [reply]
In your (root) home directory. Mill is trying to read a `build.sc` but claims, that it does not exists.

**Reply by yzjalwind** [reply]
How do i generate build.sc ？

**Reply by yzjalwind** [reply]
I currently execute  "curl -L https://github.com/com-lihaoyi/mill/releases/download/0.9.8/0.9.8 > /usr/local/bin/mill  and sh mill"

**Reply by yzjalwind** [reply]
I manually created the build.sc directory then
![image](https://user-images.githubusercontent.com/57080333/126866693-2dbd96d5-b26e-481d-93bf-6703a467e085.png)

**Reply by lefou** [reply]
`build.sc` is a file, not a directory. `ls -la` gives more details than pure `ls`. Please have a look at the mill documentation, if you have no idea what to place into your `build.sc`. But your initial command line suggested, you've already setup a project with a `chiselModule`.

https://com-lihaoyi.github.io/mill/mill/Configuring_Mill.html

**Reply by lefou** [reply]
https://com-lihaoyi.github.io/mill/mill/Intro_to_Mill.html#_getting_started
